src/clj/puppetlabs/services/jruby_pool_manager/jruby_core.clj

```diff
@@ -145,6 +149,7 @@
   [config :- {schema/Keyword schema/Any}]
   (-> config
       (update-in [:compile-mode] #(keyword (or % default-jruby-compile-mode)))
+      (update-in [:compat-version] #(str (or % default-jruby-compat-version)))
```

Comment:
Ugh, that's annoying. Yeah, I think that it should be possible to have the value in the config file be either `2.0` or `"2.0"` because I don't want to need to have to worry about whether the NC will convert the value to a string or not depending on how users enter it. @camlow325 I'll do something like your suggestion.